packages/manager/modules/octavia-load-balancer/src/create/constants.js

```diff
@@ -107,10 +107,21 @@ export const AGORA_ADDON_FAMILY = 'octavia-loadbalancer';
 
 export const SIZE_FLAVOUR_REGEX = /octavia-loadbalancer.loadbalancer-([sml]).hour.consumption/;
 export const AGORA_GATEWAY_REGEX = /gateway.s.hour.consumption/;
+export const AGORA_FLOATING_IP_REGEX = /floatingip.floatingip.hour.consumption/;
+
 export const LOAD_BALANCER_NAME_REGEX = /^[A-Za-z0-9_.-]+$/;
 
 export const NETWORK_PRIVATE_VISIBILITY = 'private';
 
+export const FLOATING_IP_TYPE = {
+  NO_IP: 'none',
+  CREATE: 'create',
+  IP: 'ip',
+};
+
+export const FLOATING_IP_CREATE_DESCRIPTION =
+  'FIP created by OVHCloud Control Panel (Manager) for Load Balancer';
```

Comment:
**question:** No translation required for this?

Reply:
Nop this is why I put this in constants